src/components/layout/CyVerseAppBar.js

```diff
@@ -289,6 +300,8 @@ function CyverseAppBar(props) {
         }
     }, [userProfile]);
 
+    useEffect(() => {}, [activeView]);
```

Comment:
I think I'm probably missing something. What does this do?

Reply:
This does nothing cos this code should have been deleted and I didn't 🤦